tests/unit/checkDeployBlockersTest.js

```diff
@@ -96,7 +96,6 @@ describe('checkDeployBlockers', () => {
             mockGetIssue.mockResolvedValue(checkedBoxesNoShipitIssue);
             // eslint-disable-next-line max-len
             baseComments.data.push({body: 'This issue either has unchecked QA steps or has not yet been stamped with a :shipit: comment. Reopening!'});
-            baseComments.data.push({body: 'This is another comment!'});
```

Comment:
I don't think I understand why this was removed?

Reply:
This line was added to update the test to work with the previous change, we want to revert the change and therefore revert the test too.

Reply:
We want to basically revert half of: https://github.com/Expensify/App/pull/4824 since it had two changes:
1. Regex for :shipit: (and the updated tests)
2. 404 issue with the wrong link

This is just reverting 1.